scripts/eosio_build_amazon.sh

```diff
@@ -183,7 +152,8 @@ if [ "$BUILD_CLANG8" = "true" ]; then
       && rm -rf ${TMP_LOCATION}/clang8 \
       && cd ../.. \
       || exit 1
-
+      export CXX=$CPP_COMP
```

Comment:
I don't think these two lines shouldn't be here. (Same for the other OSes.)

Reply:
They set CXX to use the new clang: LLVM, boost, etc, look for CXX and without it the cpp/c++ version on the machine is used instead of the new clang8.